charts/actions-runner-controller/docs/UPGRADING.md

```diff
@@ -29,6 +29,17 @@ curl -L https://github.com/actions/actions-runner-controller/releases/download/a
 kubectl replace -f crds/
 ```
 
+In case you're going to create prometheus-operator `ServiceMonitor` resources via the chart, you'd need to deploy prometheus-operator-related CRDs as well.
+
+For example, you can use the `appscode/prometheus-operator-crds` to deploy those CRDs with Helm:
+```
+$ helm repo add appscode https://charts.appscode.com/stable/
```

Comment:
which chart might be better? `prometheus-community/prometheus-operator-crds` or `appscode/prometheus-operator-crds`

Reply:
@dhawalseth - [prometheus.io](http://prometheus.io/) holds no opinion (as far as we're aware) on how to install directly into the Kubernetes cluster. As such, we think it's best if we leave this decision for our end users to make.

Is there a particular reason you're recommending this specific solution?

Reply:
No particular reason. we can leave this up to the end users.